components/google-cloud/google_cloud_components/mb_sdk_component/remote_runner.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for remote execution of Model Builder SDK as a pipeline component."""
+
+import argparse
+import inspect
+import os
+from google.cloud import aiplatform
+from google.cloud import storage
+
+INIT_KEY = 'init'
+METHOD_KEY = 'method'
+
+
+def split_args(kwargs):
+  """Split args to match MB SDK input format for init, method and config."""
```

Comment:
Looks like the correct interpretation of kwargs relies on specific convention of the keys. IMO it worths some explanation and examples. Maybe it also makes sense to assert that key is following the correct format in the conditional branches below (L33 - 38)

Reply:
Acknowledged, added a TODO to track this to add before the final product.